reference/src/layout/unions.md

```diff
@@ -18,10 +18,10 @@ overlap. This can be visualized as follows:
 [ <---> [field2_ty] <---> ]
 ```
 
```

Comment:
Not sure if the empty line here makes a difference, but I think removing it makes the source look better by gluing the caption closer to the figure.